ctapipe/core/component.py

```diff
@@ -80,3 +94,24 @@ def __init__(self, parent=None, config=None, **kwargs):
             self.log = getLogger(
                 self.__class__.__module__ + '.' + self.__class__.__name__
             )
+
+    @classmethod
```

Comment:
@MaxNoe Here the code of Component is modified (relevant to conversation in #917)

Reply:
But this is just a utility method to get a subclass by name, right?

This is not "Implementing a Factory-like" behaviour.

Reply:
this is outdated, I'll close it here.